Add MacOS installation guide (#2171)
ppy/osu-wiki: wiki/Installation/en.md

> <!-- *See also: [Installation/OS X](/wiki/Installation/OS_X) and [Installation/Linux](/wiki/Installation/Linux)* -->

Reviewer: ```suggestion
*See also: [Installation/Mac_OS](/wiki/Installation/Mac_OS)* <!-- and [Installation/Linux](/wiki/Installation/Linux)* -->
```

Reviewer 2: I think it is actually spelt as `macOS` with lowercase M and no space.

Reviewer: hasn't been addressed

> This page will roughly tell you how to get osu! working on various devices. The [osu!academy](/wiki/osu!academy) has a [video tutorial](https://www.youtube.com/watch?list=PLmWVQsxi34bMYwAawZtzuptfMmszUa_tl&v=cz522ZAs5aQ) that explains how to install osu! on Windows.

Reviewer: revert this change

> This page will roughly tell you how to get osu! working on your windows device. The [osu!academy](/wiki/osu!academy) has a [video tutorial](https://www.youtube.com/watch?list=PLmWVQsxi34bMYwAawZtzuptfMmszUa_tl&v=cz522ZAs5aQ) that explains how to install osu! on Windows.

Reviewer: capitalize Windows

5.  osu! will prompt you to sign in or register.
    -   If you have an account already, sign in.
    -   If you need to register, see [Registration](/wiki/Registration), then sign in.
    

> ### MacOS

Reviewer: move this section to a `wiki/Installation/Mac_OS/en.md article 

https://gist.github.com/TPGPL/1beea5d9293dfe5e5781f8e29060d2e4 - article's content

> - Since the MacOS version of osu! is unofficial, the game may not fuction properly, or even boot at all. 

Reviewer 2: ```suggestion
The MacOS version of osu! is experimental. The game may not function properly or even boot at all. 
```

1.  Go to the unofficial [download page](https://osu.ppy.sh/community/forums/topics/682197) and download the installer.
2.  Locate the installer, right click, and select open.

>     - Double left clicking on the file may not open it, since MacOS may recognize the app as developed by an unidentified devloper. Right clicking and selecting open will solve this problem.

Reviewer 2: ```suggestion
    -   Double left clicking on the file may not open it, since MacOS may recognize the app as developed by an unidentified developer. Right clicking and selecting open will solve this problem.
```

Reviewer 3: was the space at the front intentional

Reviewer 2: Yes, it would keep the spacing consistent with the numbered list.

> 3.  The installer will automatically run. You cannot change the installation location, since it is installed directly to `osu!.app`. Moving `osu.app` will move all of the files the come with it.

Reviewer 2: ```suggestion
3.  The installer will automatically run. The installation location cannot be changed. It will be installed directly to `osu!.app`. Moving `osu.app` will move all of the files the come with it.
```
